app/resources/css-default.template

```diff
@@ -607,6 +612,16 @@ td.submit {
   line-height: 10px;
 }
 
+@media (min-width: 700px) {
+  .main .action-wrapper {
+    display: inline-block;
+  }
+  .main .action-outer {
+    float: left;
```

Comment:
The goal here is to layout horizontally instead of virtually, right? Can't you use inline-block here too?

Reply:
Yep, done.